.github/workflows/benchmarks.yml

```diff
@@ -12,8 +12,6 @@ jobs:
 
     strategy:
       matrix:
-        dependencies:
-          - "locked"
```

Comment:
The same reasoning for `os` should go for `dependencies` too: the goal is to copy-paste https://github.com/lcobucci/jwt/pull/619/files#diff-ea12f60188cdd90bc99e5d0af2eb91647bbe4a9199176aa1ec5240f65efed510R45 everywhere

Moreover, explicit matrix is easier to debug than implicit ones

Reply:
Note: most of the matrix is copied from my repos, where I really like this explicitness too 👍

Reply:
Well, if you think it's better, let's do it 😊 
Should we keep unused steps (which depends on `dependencies` like [this](https://github.com/lcobucci/jwt/pull/619/files#diff-dfd15ae70247bb45f121522ec2e50e5da9aed17e23116d94ec887a4769c44294L43)) ?

Reply:
IMHO, yes